### PR TITLE
[core] Add mesh container for point2D

### PR DESCRIPTION
**Pull request by josep-m-carbonell**
Adding a missing mesh container in order to write elements with 2D point geometries.

### CODE REVIEW

**Review by lluis-mv — DISMISSED** [review]
perfect!

File: kratos/includes/gid_io.h [hunk 1]
```diff
@@ -198,6 +198,9 @@ class GidIO : public IO
         mGidMeshContainers.push_back( TMeshContainer(
                                           GeometryData::Kratos_Point3D,
                                           GiD_Point, "Kratos_Point3D_Mesh" ) );
+        mGidMeshContainers.push_back( TMeshContainer(
```
Comment (loumalouomega) [inline kratos/includes/gid_io.h]: Can we put the 2D first?, my OCD is tingling

imagen

Comment (josep-m-carbonell) [reply kratos/includes/gid_io.h]: Ok, no problem

**Review by loumalouomega — APPROVED** [review]
:D

**Pull request merged**